mkdocs_static_i18n/plugin.py

```diff
@@ -124,6 +125,27 @@ def on_template_context(self, context, template_name, config):
         context["i18n_alternates"] = self.i18n_files_per_language
         return context
 
+    @plugins.event_priority(50)
+    def on_page_markdown(self, markdown, page, config, files):
+        """
+        The page_markdown event is called after the page's markdown is loaded from file
+        and can be used to alter the Markdown source text.
+
+        Here we translate admonition types
+        """
+        admonition_translations = self.current_language_config.admonition_translations or {}
+        out = []
+        for line in markdown.splitlines():
+            m = re.match(r'^(!!!\s+)([^\s]+)(.*)$', line)
```

Comment:
Defining admonitions as `!!!tip` is also valid and I think that this regex does not match it.

If I'm not mistaken, please add tests for this behavior too!

Reply:
That's a good point! I haven't seen it in documentation, but you're right. So I've copied the regex from here now:
https://github.com/Python-Markdown/markdown/blob/master/markdown/extensions/admonition.py#L53